src/components/state-details.js

```diff
@@ -94,28 +94,251 @@ const getPlacesData = (data) => {
   const placesData = getPlacesData(data)
-  const emissions= emissions_2018
 
-  const [placeData, setPlaceData] = useState(placesData[currentPlace])
+  const currPlaceData = placesData[currentPlace]
+
+  // Get the last year of emissions data we have to use for showing the
+  // breakdown of how much emission comes from each source in this state
+  const placeEmissions = currPlaceData.emissions[currPlaceData.emissions.length - 1]
+
+  const totalLatestEmissions = placeEmissions.buildings +
+    placeEmissions.dirty_power +
+    placeEmissions.dumps_farms_industrial_other +
+    placeEmissions.transportation
+
+  const buildingsPrcnt = (placeEmissions.buildings / totalLatestEmissions * 100).toFixed(1)
+  const powerPrcnt = (placeEmissions.dirty_power / totalLatestEmissions * 100).toFixed(1)
+  const transportPrcnt = (placeEmissions.transportation / totalLatestEmissions * 100).toFixed(1)
+
+  const placeTitle = currPlaceData.name
+
+  const [placeData, setPlaceData] = useState(currPlaceData)
 
   return (
     <Layout>
       <SEO />
 
-      <div className='row d-flex flex-row'>
-        <div className='col-12 col-lg-4'>
-          <h1 className='mr-4 mb-3'>{placeData.name}</h1>
-          <ChoroplethMap emissions={emissions} sidebar={false} selected_location={currentPlace}/>
+      <div className='col-6'>
+        <h1 className='mr-4 mb-3'>{placeData.name}</h1>
+        <ChoroplethMap emissions={countryEmissions} sidebar={false} selected_location={currentPlace}/>
+      </div>
+
+      {/* Intro Section */}
+      <div className='col-12'>
+        <p className="h1 mt-5 mb-5">
+          To get to <strong>zero</strong> by 2050, {placeTitle} must
+          cut climate pollution by <strong>{cutPerYearPrcnt}% a year.</strong>
+        </p>
+
+        <p className="h4 font-weight-bold">Emissions in {placeTitle}</p>
+        <p className="h6 text-muted">
+          Metric tons of carbon dioxide equivalent (MTCO2e) emissions
+        </p>
+        <StackedBarChart emissions_data={placeData.emissions}/>
+
+        <p className="h1 text-center mt-5">We can do it. Here's how.</p>
+
+        <hr className="mt-5"/>
+      </div>
+
+      {/* Buildings Section */}
+      <div className='col-12'>
+        <h2 className="h4 mt-4 ml-5 font-weight-bold">Buildings</h2>
+
+        <p className="h3 mt-5">
+          <strong>{buildingsPrcnt}%</strong> of emissions in {placeTitle} comes from buildings.
+        </p>
+
+        <p className="h4 mt-5 text-muted">
+          [insert state emissions graph highlighting buildings]
+        </p>
+
+        <p>
+          Mostly from heating them.
+
+          ?% of the pollution of your typical home comes from heating your space, water, and food.
+        </p>
+
+        <p className="h3 mt-5">
+          To stop this pollution, we need to electrify our furnaces, water boilers, and stoves.
+        </p>
+
+        <p className="h3 mt-5">
+          And we need to do this for all ? buildings in {placeTitle}<br/>
```

Comment:
I noted all missing data with a ?. If you have a suggestion for something clearer, let me know! Luckily we don't actually use a ? otherwise on this page so it works out.

Reply:
I think that's fine